test/jdk/java/security/testlibrary/SimpleOCSPServer.java

```diff
@@ -335,11 +336,11 @@ private static String dumpHexBytes(byte[] data) {
      *
      * @return The hexdump of the byte array
      */
-    private static String dumpHexBytes(byte[] data, int itemsPerLine,
-            String lineDelim, String itemDelim) {
+    private static String dumpHexBytes(byte[] data, int dataLen,
+            int itemsPerLine, String lineDelim, String itemDelim) {
```

Comment:
Yes, I would prefer to keep this.  I made the change in order to help me debug draining the input stream and I felt it prudent to leave it in place, along with the main test's debug flag just in case.